script/bootstrap

```diff
@@ -15,3 +15,6 @@ if [ -f .env.${NODE_ENV} ]; then
 fi
 
 npm install
+
+# Make sure mocha and coffee are in the PATH
+export PATH="node_modules/.bin:$PATH"
```

Comment:
Is this file not intended to always be sourced immediately prior to running tests? I was working under the assumption you would always run `script/test`, which sources this file.

I think the specific path will break if mocha is installed globally instead of within the local directory. I'll test that out to be sure though.